src/eth/EthereumCdpService.js

```diff
@@ -440,21 +440,22 @@ export default class EthereumCdpService extends PrivateService {
     );
   }
 
-  async wipeDaiProxy(dsProxyAddress, cdpId, amount, useOtc = false) {
+  @tracksTransactions
+  async wipeDaiProxy(dsProxyAddress, cdpId, amount, { useOtc, promise }) {
```

Comment:
@michaelelliot just a heads-up: I changed the method signature here and on `shutProxy` below